diff -uNdr GG.original/src/adobe/any_regular.cpp GG/src/adobe/any_regular.cpp
--- GG.original/src/adobe/any_regular.cpp	2011-07-26 21:42:27.000000000 +0400
+++ GG/src/adobe/any_regular.cpp	2011-07-26 22:39:26.000000000 +0400
@@ -52,6 +52,7 @@
 
 template <typename T>
 struct serializable : serializable_t {
+    serializable() {} ;
     void operator () (std::ostream& out, const any_regular_t& x) const
     { out << format(x.cast<T>()); }
 };
