diff -uNdr GG.original/CMakeLists.txt GG/CMakeLists.txt
--- GG.original/CMakeLists.txt	2011-07-26 21:42:30.000000000 +0400
+++ GG/CMakeLists.txt	2011-07-26 23:42:47.858218933 +0400
@@ -320,20 +320,11 @@
             COMMAND ${DOXYGEN_EXECUTABLE}
             WORKING_DIRECTORY ${CMAKE_HOME_DIRECTORY}/doc
         )
-        execute_process(
-            COMMAND ${CMAKE_COMMAND} -E copy_directory
-            ${CMAKE_HOME_DIRECTORY}/doc/GG
-            ${CMAKE_BINARY_DIR}/doc/GG
-        )
-        execute_process(
-            COMMAND ${CMAKE_COMMAND} -E remove_directory
-            ${CMAKE_HOME_DIRECTORY}/doc/GG
-        )
     endif ()
     if (BUILD_DOCUMENTATION AND NOT NO_DOCS_PACKAGE)
         install(
-            DIRECTORY ${CMAKE_BINARY_DIR}/doc/GG
-            DESTINATION doc
+            DIRECTORY ${CMAKE_BINARY_DIR}/doc/libgigi-${GIGI_VERSION}
+            DESTINATION ${SHARE_INSTALL_PREFIX}/doc
         )
     endif ()
 elseif (BUILD_DOCUMENTATION)
diff -uNdr GG.original/doc/Doxyfile GG/doc/Doxyfile
--- GG.original/doc/Doxyfile	2011-07-26 21:42:30.000000000 +0400
+++ GG/doc/Doxyfile	2011-07-26 23:45:12.930569039 +0400
@@ -748,7 +748,7 @@
 # If a relative path is entered the value of OUTPUT_DIRECTORY will be 
 # put in front of it. If left blank `html' will be used as the default path.
 
-HTML_OUTPUT            = GG
+HTML_OUTPUT            = libgigi-0.8.0
 
 # The HTML_FILE_EXTENSION tag can be used to specify the file extension for 
 # each generated HTML page (for example: .htm,.php,.asp). If it is left blank 
diff -uNdr GG.original/GG/adobe/cmath.hpp GG/GG/adobe/cmath.hpp
--- GG.original/GG/adobe/cmath.hpp	2011-07-26 21:42:30.000000000 +0400
+++ GG/GG/adobe/cmath.hpp	2011-07-26 23:18:56.832876049 +0400
@@ -37,7 +37,7 @@
 
 #define ADOBE_HAS_CPP_CMATH 
 
-#elif ((__GNUC__ == 4) && (__GNUC_MINOR__ <= 5))
+#elif ((__GNUC__ == 4) && (__GNUC_MINOR__ <= 6))
 /*
     The currently supported version of GNUC has C99 extensions in math.h. But no TR1 extensions.
 */
diff -uNdr GG.original/src/adobe/any_regular.cpp GG/src/adobe/any_regular.cpp
--- GG.original/src/adobe/any_regular.cpp	2011-07-26 21:42:27.000000000 +0400
+++ GG/src/adobe/any_regular.cpp	2011-07-26 22:39:26.000000000 +0400
@@ -52,6 +52,7 @@
 
 template <typename T>
 struct serializable : serializable_t {
+    serializable() {} ;
     void operator () (std::ostream& out, const any_regular_t& x) const
     { out << format(x.cast<T>()); }
 };
diff -uNdr GG.original/src/GIL/extension/dynamic_image/apply_operation_base.hpp GG/src/GIL/extension/dynamic_image/apply_operation_base.hpp
--- GG.original/src/GIL/extension/dynamic_image/apply_operation_base.hpp	2011-07-26 21:42:28.000000000 +0400
+++ GG/src/GIL/extension/dynamic_image/apply_operation_base.hpp	2011-07-26 22:24:02.000000000 +0400
@@ -118,7 +118,7 @@
     template <typename T2, typename Op>
     struct reduce_bind1 {
         const T2& _t2;
-        mutable Op&  _op;
+        Op&  _op;
 
         typedef typename Op::result_type result_type;
 
@@ -131,7 +131,7 @@
     struct reduce_bind2 {
         const Bits1& _bits1;
         std::size_t _index1;
-        mutable Op&  _op;
+        Op&  _op;
 
         typedef typename Op::result_type result_type;
 
